pdr_backend/models/erc721.py

```diff
@@ -0,0 +1,25 @@
+from typing import Union
```

Comment:
Q: should this be called erc721.py (class ERC721) or data_nft.py (class DataNft)?

Investigate: well what does it _do_ here?

Result: it's calling `addToCreateERC20List`. DataNft has this functionality, but canonical ERC721 does not.

Answer to Q: call it data_nft.py (class DataNft)